scripts/configure

```diff
@@ -26,98 +14,194 @@ check_dependencies () {
+  echo "You may work around this by modifying the configuration script yourself, but it's highly experimental."
+  echo "If you get it working, we hope you consider making a PR. :)"
+  exit 1	
+fi
 
-UMBREL_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))/.."
+check_dependencies docker docker-compose dirname readlink
+
+# Get gnu sed
+gnused=sed
+if [[ "$(uname)" == "Darwin" ]]; then
+  if ! command -v gsed >/dev/null 2>&1; then
+    echo "Error: This script requires gnu-sed!"
+    echo "Install it with:"
+    echo "  brew install gnu-sed"
+    exit 1
+  fi
+  gnused=gsed
+fi
 
+# Switch to Umbrel's root directory
+UMBREL_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))/.."
 if [[ ! -d "$UMBREL_ROOT" ]]; then
   echo "Root dir does not exist '$UMBREL_ROOT'"
   exit 1
 fi
-
 cd "$UMBREL_ROOT"
 
-echo "Start box configuration"
-echo "Installing RPCAuth.py and configuring secrets"
-cd bin/
-wget -q "https://raw.githubusercontent.com/bitcoin/bitcoin/master/share/rpcauth/rpcauth.py" 2>/dev/null
-chmod 755 rpcauth.py
-./rpcauth.py lncm | tee ../secrets/generated.txt | head -2 | tail -1 > ../secrets/rpcauth.txt
-tail -1 ../secrets/generated.txt > ../secrets/rpcpass.txt
-rm rpcauth.py ../secrets/generated.txt
-cd ..
-echo "Installing RPCAuth into bitcoin.conf"
-cat secrets/rpcauth.txt >> bitcoin/bitcoin.conf
-RPCPASS=`cat secrets/rpcpass.txt`
-echo "Configuring LND rpc info"
-sed -i "s/RPCPASS/${RPCPASS}/g; " lnd/lnd.conf
-echo "Configuring docker-compose file"
-sed -i "s/RPCPASS/${RPCPASS}/g; " docker-compose.yml
-# TESTNET set and REGTEST not
-if [ ! -z $TESTNET ] && [ -z $REGTEST ]; then
-    echo "Enabling testnet mode if TESTNET variable is set"
-    # Update bitcoin.conf
-    sed -i 's/\#\[test\]/\[test\]/g;' bitcoin/bitcoin.conf
-    sed -i 's/\#testnet=1/testnet=1/g' bitcoin/bitcoin.conf
-    sed -i 's/rpcport=8332/rpcport=18332/g; ' bitcoin/bitcoin.conf
-    sed -i 's/port=8332/port=18333/g; ' bitcoin/bitcoin.conf
-    echo "Setting testnet port"
-    sed -i 's/RPCPORT/18332/g; ' docker-compose.yml
-    # Update docker-compose
-    sed -i 's/mainnet/testnet/g; ' docker-compose.yml
-    # lnd.conf
-    echo "Changing LND to testnet mode"
-    sed -i 's/bitcoin.mainnet=1/bitcoin.testnet=1/g; ' lnd/lnd.conf
-    echo "Updating LND neutrino peers"
-    sed -i 's/neutrino.addpeer=bb2.breez.technology/\;neutrino.addpeer=bb2.breez.technology/g; ' lnd/lnd.conf
-    sed -i 's/neutrino.addpeer=mainnet1-btcd.zaphq.io/\;neutrino.addpeer=mainnet1-btcd.zaphq.io/g; ' lnd/lnd.conf
-    sed -i 's/neutrino.addpeer=mainnet2-btcd.zaphq.io/\;neutrino.addpeer=mainnet2-btcd.zaphq.io/g;' lnd/lnd.conf
-    sed -i 's/\;neutrino.addpeer=testnet1-btcd.zaphq.io/neutrino.addpeer=testnet1-btcd.zaphq.io/g;' lnd/lnd.conf
-    sed -i 's/\;neutrino.addpeer=testnet2-btcd.zaphq.io/neutrino.addpeer=testnet2-btcd.zaphq.io/g; ' lnd/lnd.conf
+# Configure for mainnet or testnet or regtest depending
+# upon the user-supplied value of $NETWORK
+BITCOIN_CHAIN="${NETWORK:-mainnet}"
```

Comment:
Yeah, I agree. `$BITCOIN_NETWORK` sounds more apt than `$BITCOIN_CHAIN`. Updated! I originally went with `chain` because that's also how it's referred by `bitcoin-cli getblockchaininfo` too.